### DISCUSSION TITLE
Next.js Local Production Environment Incorrectly Reading .env.development Instead of .env.production on Windows

### DISCUSSION THREAD

**Discussion by ekiyak** [vercel/next.js#65794] [App Router]
I'm testing the production settings of my project locally before deploying it to the server. I perform the build process, and when I run the application, I can observe the process.env value as production through console.log. However, my server-side or client-side code still reads settings from .env.development instead of the production environment file, which I can verify by logging the API base URL. I am working on a Windows environment and have set up my script commands as follows:

`"dev": "set NODE_ENV=development&& next dev",
"build": "set NODE_ENV=production&& next build",
"start": "set NODE_ENV=production&& next start"
`

How can I ensure that my application correctly reads from the .env.production file when running in production mode locally?"

**Comment by icyJoseph** [comment]
The NODE_ENV variable is already set by Next.js, you don't have to set it yourself, but that wouldn't explain the behavior your describe, though it might be a good idea to try and remove the `set NODE_ENV` bits, since they are not needed. 

Also remove the `.next` folder before doing a build.
